<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModificarCampo</class>
 <widget class="QDialog" name="DialogModificarCampo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="T_ModificarNombre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modificar Campo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitud del Campo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModificarCampo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Campo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_ModificarLlave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Llave Primaria</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SB_ModificarLongitud">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>163</y>
     <width>48</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_ModificarTipo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CHAR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INTF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DEC</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
